akka-persistence/src/main/scala/akka/persistence/fsm/PersistentFSMActor.scala

```diff
@@ -17,9 +17,12 @@ import scala.reflect.ClassTag
  * FSM actor implementation with persistent state
  *
  * Supports the usual [[akka.actor.FSM]] functionality with additional persistence features.
- * State and State Data are persisted on every state change.
- * FSM is identified by 'persistenceId' value.
+ * State changes are persisted. FSM is identified by 'persistenceId' value.
```

Comment:
Maybe: `State changes are persisted on every state change (except when `stay` is used)`